pkgs/by-name/go/goutline/package.nix

```
@@ -0,0 +1,35 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  pkgs,
+}:
+
+buildGoModule rec {
+  pname = "goutline";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "1pkg";
+    repo = "goutline";
+    rev = "v${version}";
+    hash = "sha256-YZM+pQobuFGlxX+t1rgiM8JMc4n1AWveQr0W3LE9yPc=";
+    postFetch = ''
+      export GOCACHE=$TMPDIR/go-build
+      export GOPATH=$TMPDIR/go
+      export GOMODCACHE=$TMPDIR/go/pkg/mod
+      cd $out
+      ${pkgs.go}/bin/go mod init github.com/1pkg/goutline
+      ${pkgs.go}/bin/go mod tidy
```

This is the only place in Nixpkgs where I could find a need for this, so I suspect other places that might have done this instead used a patch or upstreamed a change to the package that added go module support.

Nonetheless, since you've got this working this way, you could refer to the executable of the `go` tool like this:
```suggestion
      ${lib.getExe go} mod init github.com/1pkg/goutline
      ${lib.getExe go} mod tidy
```

---

Thank you, fixed!